spec/crypto/hashing_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Hashing do
+  let(:password) { 'foo password' }
+
+  it 'should create SHA512 password hash' do
+    hash = Hashing.hash(password)
+    expect(hash).to match(/^sha512\$.*\$.*/)
```

Comment:
I find it quite important to check if the generated hash is working correctly.

how about moving https://github.com/puzzle/cryptopus/blob/master/app/models/user.rb#L66 also to Crypto::Hashing ?
#matches?(salted_hash, password)